samples/bluetooth/mesh/light_switch/overlay-lpn.conf

```diff
@@ -0,0 +1,30 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+CONFIG_BT_MESH_LOW_POWER=y
+CONFIG_BT_DEVICE_NAME="Mesh Light Switch LPN"
+CONFIG_BT_MESH_RELAY=n
+CONFIG_BT_MESH_FRIEND=n
+
+# Power saving configuration:
+
+# Serial communication consumes considerable power
+CONFIG_SERIAL=n
+CONFIG_UART_CONSOLE=n
+
+# While enabled, secure beacons will be advertised periodically.
+# This consumes power, and is not required for a LPN node.
+CONFIG_BT_MESH_BEACON_ENABLED=n
+
+# Each LPN poll event consumes power. Extending the intraval between poll events will improve power consumption
+CONFIG_BT_MESH_LPN_POLL_TIMEOUT=600
```

Comment:
Is this sensible for a real life light switch application? I guess the light switch doesn't really need timely updates from the network, it will only receive statuses and not really change behavior as a result of these updates, so it's probably fine, just wondering.